package.json

```diff
@@ -46,12 +46,16 @@
     "classnames": "^2.3.2",
     "react-bootstrap-typeahead": "^6.1.2",
     "react-datepicker": "^4.11.0",
+    "react-jsx-runtime": "^1.0.0-alpha.1",
```

Comment:
@ita-alibb 

Is this needed? Seems odd to install a alpha package?

Reply:
No, my bad during installation! Than you